evidence/create_release_bundle.go

```diff
@@ -64,23 +70,34 @@ func (c *createEvidenceReleaseBundle) Run() error {
 
 func (c *createEvidenceReleaseBundle) buildReleaseBundleSubjectPath(artifactoryClient artifactory.ArtifactoryServicesManager) (string, string, error) {
 	repoKey := buildRepoKey(c.project)
-	manifestPath := buildManifestPath(repoKey, c.releaseBundle, c.releaseBundleVersion)
-
-	manifestChecksum, err := c.getFileChecksum(manifestPath, artifactoryClient)
+	manifestPath, manifestChecksum, err := c.buildReleaseBundleSubjectPathWithManifestName(artifactoryClient, releaseBundleEvdManifestName, repoKey)
+	if err == nil {
+		return manifestPath, manifestChecksum, nil
+	}
+	// fallback to legacy name
+	manifestPath, manifestChecksum, err = c.buildReleaseBundleSubjectPathWithManifestName(artifactoryClient, releaseBundleEvdManifestNameLegacy, repoKey)
```

Comment:
please log the error as info or at least debug and mention falling back to legacy name